projects/laji/src/app/+observation/form/media-filter/media-filter.component.ts

```diff
@@ -0,0 +1,65 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { WarehouseQueryInterface } from '../../../shared/model/WarehouseQueryInterface';
+
+const mediaGroupProps = ['hasUnitImages', 'hasUnitAudio', 'hasUnitModel'] as const;
```

Comment:
This could have a type that is linked with `WarehouseQueryInterface`
```
type MediaGroupProp = keyof Pick<WarehouseQueryInterface, 'hasUnitImages'|'hasUnitAudio'|'hasUnitModel'>;
const mediaGroupProps: MediaGroupProp[] = ...
```
Then it would throw an error if some of the props had a typo (I guess this doesn't do it?). But not that important thing